src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the literal '7' with a constant that explains its meaning, such as 'LONG_TERM_RENTAL_DAYS'. Similarly, replace '40' with a constant like 'DAILY_RENTAL_RATE' and '50' with 'LONG_TERM_DISCOUNT'.